src/1729Essay.sol

```diff
@@ -1,36 +1,125 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: MIT
 pragma solidity ^0.8.15;
 
-import {ERC721} from "solmate/tokens/ERC721.sol";
-
-struct Essay {
-    uint8 cohort;
-    uint8 week;
-    uint16 voteCount; // NOTE moved to here from end for more efficient storage
-    string name;
-    string image;
-    string description;
-    string contentHash; // TODO switch to fixed-size bytes array
-    string writerName;
-    string writerAddress; // TODO switch to address
-    string publicationURL;
-    string archivalURL;
-}
+//import {IERC2981} from "openzeppelin-contracts/contracts/interfaces/IERC2981.sol";
+import {ERC721} from "openzeppelin-contracts/token/ERC721/ERC721.sol";
+import {ERC721URIStorage} from "openzeppelin-contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import {ERC2981} from "openzeppelin-contracts/token/common/ERC2981.sol";
+import {Ownable} from "openzeppelin-contracts/access/Ownable.sol";
+//import "openzeppelin-contracts/contracts/utils/math/SafeMath.sol";
+//import {IERC165} from "openzeppelin-contracts/utils/introspection/IERC165.sol";
+
+////////////////////////////////////////////////////////////////////////////
+//                                                                        //
+//         _ _____ ____   ___    __    __      _ _                        //
+//        / |___  |___ \ / _ \  / / /\ \ \_ __(_) |_ ___ _ __ ___         //
+//        | |  / /  __) | (_) | \ \/  \/ / '__| | __/ _ \ '__/ __|        //
+//        | | / /  / __/ \__, |  \  /\  /| |  | | ||  __/ |  \__ \        //
+//        |_|/_/  |_____|  /_/    \/  \/ |_|  |_|\__\___|_|  |___/        //
+//                                                                        //
+////////////////////////////////////////////////////////////////////////////
 
-/// @title An essay from the 1729 writers union
+/// @title A collection of winning essays from 1729Writers
 /// @author neodaoist, plaird
-/// @notice A 1729w admin can mint and burn essay NFTs on this contract
+/// @notice A 1729Writers admin can mint and burn essay NFTs on this contract
 /// @dev XYZ
-contract OneSevenTwoNineEssay is ERC721 {
+contract OneSevenTwoNineEssay is Ownable, ERC721, ERC2981 {
+    //using SafeMath for uint256;
     //
-    constructor() ERC721("1729 Essay", "1729ESSAY") {}
+
+    struct EssayItem {
+        address author;
+        string url;
+    }
+
+    mapping(uint256 => EssayItem) public essays;
+
+    constructor(address _multisig) ERC721("1729 Essay", "1729ESSAY") {
+        transferOwnership(_multisig);
```

Comment:
We may want to use the internal function _transferOwnership instead. Not blocking